content/en/profiler/profiler_troubleshooting/dotnet.md

```diff
@@ -72,6 +72,73 @@ If you've configured the profiler and don't see profiles in the profile search p
 
    Note that the following message could appear, but it does not impact Datadog profiling: `Profiler signal handler has been replaced. Restoring it.` This indicates only that the Datadog signal handler is reinstalled when it was overwritten.
 
+5. Check that the following environment variables `CORECLR_ENABLE_PROFILING`, `CORECLR_PROFILER`, `CORECLR_PROFILER_PATH` are not set, and that the value pointing to `Datadog.Linux.ApiWrapper.x64.so` in `LD_PRELOAD` is not set. If this is the case, remove them from scripts or from the docker file where they are set. Otherwise, an old manually installed version of the profiler will still be used.
+
+6. Check that the value of `DD_PROFILING_ENABLED` is set to `1` or `Auto`. For the latter, the profiler will generate profiles if and only if (1) a trace was created and (2) the application ran for more than 30 seconds.
+If it is set to another value or not set at all, the profiler is disabled.
+
+[1]: /profiler/enabling/dotnet/?tab=linux#configuration
+
+{{% /tab %}}
+{{% tab "Linux" %}}
+
+1. Check that the Agent is installed and running.
+
+2. Check that the profiler has been loaded from the loader log:
+
+   1. Open the `dotnet-native-loader-dotnet-<pid>` log file in the `/var/log/datadog` folder.
+
+   2. Look for `CorProfiler::Initialize: Continuous Profiler initialized successfully.` near the end. If that message is not present, enable debug logs by setting the `DD_TRACE_DEBUG` environment variable for the application.
+
+   3. Restart the application.
+
+   4. Open the `dotnet-native-loader-dotnet-<pid>` log file in the `/var/log/datadog` folder.
+
+   5. Look for the `#Profiler` entry.
+
+   6. Check the following lines to ensure that the profiler library has been loaded:
+      ```
+      [...] #Profiler
+      [...] PROFILER;{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A};win-x64;.\Datadog.Profiler.Native.dll
+      [...] PROFILER;{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A};win-x86;.\Datadog.Profiler.Native.dll
+      [...] DynamicDispatcherImpl::LoadConfiguration: [PROFILER] Loading: .\Datadog.Profiler.Native.dll [AbsolutePath=/opt/datadog/linux-x64/./Datadog.Tracer.Native.so]
+      [...] [PROFILER] Creating a new DynamicInstance object
+      [...] Load: /opt/datadog/linux-x64/./Datadog.Tracer.Native.so
+      [...] GetFunction: DllGetClassObject
+      [...] GetFunction: DllCanUnloadNow
+      ```
+
+3. Check that the value of `DD_PROFILING_ENABLED` is set to `1`; if it is not set or with another value, the profiler is disabled.
```

Comment:
minor, it could have `auto`.

Reply:
`Auto` should ONLY be set with SSI: this is why I did not mention it in this Linux tab